setup.py

```diff
@@ -68,6 +68,7 @@
         "prettytable>=3.0.0",
         "pycep-parser==0.3.7",
         "charset-normalizer",
+        "pyston_lite_autoload==2.3.4.1; python_version=='3.8' and (sys_platform=='linux' or sys_platform=='darwin')"
```

Comment:
@ayajbara Why lock it to version 3.8?

Reply:
1) what will happen in case the user has "windows"? is checkov expected to installed anyway, but without "pyston_lite_autoload"?
2) why do you lock `pyston_lite_autoload==2.3.4.1`, and not `pyston_lite_autoload>=2.3.4.1` as in the Pipfile?
3) in case pytson_lite works only for python 3.8 with mac/linux, shouldn't you add condition in Pipfile?

Reply:
1. yes, it will be just without the package
2. & 3. good idea to keep it consistent between both files. Actually the `Pipfile.lock` is only used for local development. the `setup.py` is the important one, when users install it via `pypi`

Reply:
@ayajbara should be possible to add it https://pipenv-fork.readthedocs.io/en/latest/advanced.html#specifying-basically-anything

Reply:
@gruebel updated, thanks!!

Reply:
cool!!